core/src/main/scala/sttp/tapir/model/ServerRequest.scala

```diff
@@ -16,6 +17,7 @@ trait ServerRequest extends RequestMetadata {
   def queryParameters: QueryParams
 
   lazy val acceptsContentTypes: Either[String, Seq[ContentTypeRange]] = Accepts.parse(headers)
+  val timestamp: Deadline = Deadline.now
```

Comment:
is that the right type to capture a timestamp?

Reply:
maybe let's call it simply `requestStart`? It's not clear what this timestamp is

Reply:
It states that it main purpose is for repeated attempts, but on the other hand it's has `now` which captures current nanoseconds time, and +/- operators which is handy for getting time diff.
I think most appropriate type here would  be plain `Long` from `System.nanoTime` but it carries no time context, so maybe `Duration` ?